projects/1.20/assets/biomancy/biomancy/lang/zh_cn.json

```
@@ -0,0 +1,497 @@
+	"item.biomancy.gem_fragments": "宝石碎块",
+	"item.biomancy.gem_fragments.tooltip": "装点你的血肉造物！\n用于增加硬化纤维的最大弹性，十分贵重",
+	"item.biomancy.bio_lumens": "生物荧光黏液",
+	"item.biomancy.bio_lumens.tooltip": "一种基础成分，通常在分解器里和生物质一起产出\n\n能够让物品闪闪发光",
+	"item.biomancy.organic_matter": "生物质",
+	"item.biomancy.organic_matter.tooltip": "从柔软的植物嫩叶里提取出来的物质，其棕褐色的外观使其看起来和腐殖质无二",
+	"item.biomancy.exotic_dust": "怪异尘埃",
+	"item.biomancy.exotic_dust.tooltip": "奇怪的物质，没准儿真的有些蹊跷\n在分解器里分解一些不属于主世界的东西时可以获得",
+	"item.biomancy.stone_powder": "碎石粉",
+	"item.biomancy.stone_powder.tooltip": "极其基础的物质成分，通常以副产品的身份游走于各种合成配方里",
+	"item.biomancy.nutrients": "营养素",
+	"item.biomancy.nutrients.tooltip": "硬的和石头似的的富含营养的小块，和维生素片差不多",
+	"item.biomancy.nutrient_paste": "营养膏",
+	"item.biomancy.nutrient_paste.tooltip": "营养素和生物质糅合而成的膏状物\n看起来有些像是块卖相不是很好的蛋糕，是各种活体机械的优质能量来源",
+	"item.biomancy.nutrient_bar": "营养棒",
+	"item.biomancy.nutrient_bar.tooltip": "把营养膏压缩成一整块食物棒后的样子。能吃，但没什么味道",
+	"item.biomancy.bloomberry": "蕾莓",
+	"item.biomancy.bloomberry.tooltip": "怪异而美味的果实，带有荧荧微光，食用时给予随机药水效果",
+	"item.biomancy.regenerative_fluid": "再生黏液",
+	"item.biomancy.regenerative_fluid.tooltip": "有着极强再生能力的液体，可以用于调制各种原液",
+	"item.biomancy.withering_ooze": "凋灵软泥",
+	"item.biomancy.withering_ooze.tooltip": "一种腐蚀性的提取物。看起来可以用于活体酿造。",
+	"item.biomancy.hormone_secretion": "内分泌激素",
+	"item.biomancy.hormone_secretion.tooltip": "富含各种让人亢奋的激素的提取物，我想它并不应该被滥用",
+	"item.biomancy.toxin_extract": "毒素提取物",
+	"item.biomancy.toxin_extract.tooltip": "一种从各种方面来说都有剧毒的物质，极其适合制作各种可疑物质",
+	"item.biomancy.volatile_fluid": "不稳定流体",
+	"item.biomancy.volatile_fluid.tooltip": "一种易燃易爆的提取物，然而它还可被进一步加工成真正的危险品。",
+	"item.biomancy.bile": "胆汁",
+	"item.biomancy.bile.tooltip": "活体酿造时经常会用到的东西",
+	"item.biomancy.despoil_sickle": "血肉掠夺镰",
+	"item.biomancy.despoil_sickle.tooltip": "十分脆弱的武器，却极擅长从濒死生物中掠夺出特殊的战利品\n\n手持时，此工具必定能产出凌迟战利品，同时额外消耗耐久度",
+	"item.biomancy.toxicus": "[WIP] 剧毒长刀",
+	"item.biomancy.toxicus.tooltip": "[PH] 造成腐蚀伤害",
+	"item.biomancy.vial": "有机试管",
+	"item.biomancy.vial.tooltip": "一种小型有机容器，有着极强的炼金抗性。作为容纳各种活性原液的容器实在是太合适不过了\n会在使用时溶化",
+	"item.biomancy.living_flesh": "鲜活血肉",
+	"item.biomancy.living_flesh.tooltip": "毫无疑问它是活着的！\n但是对于真正意义上的生物来说，它实在是太愚昧了……也许作为一台机器的运转核心倒是不错的选择",
+	"item.biomancy.primordial_core": "原初核心",
+	"item.biomancy.primordial_core.tooltip": "带有不祥气息的物件，全由血肉构成；看着就让人有些不安……",
+	"item.biomancy.guide_book": "[WIP] 原初的躁动",
+	"item.biomancy.guide_book.tooltip": "[WIP] 这是怎样的企图？",
+	"item.biomancy.creator_mix": "怪异的血肉混合物",
+	"item.biomancy.creator_mix.tooltip": "为血肉温床准备的一顿大餐……我没说这是你可以吃的",
+	"item.biomancy.injector": "活体注射器",
+	"item.biomancy.injector.tooltip": "一种简单的生物器械，使用尖锐而中空的骨刺将调制的原液迅猛地注入进生物或者玩家体内\n\n接受以下魔咒：\n - 穿透：增加穿透盔甲的概率\n - 麻醉：取消针头造成的伤害",
+	"item.biomancy.fertilizer": "生物炼化肥料",
+	"item.biomancy.fertilizer.tooltip": "使植物疯长的奇异混合物——对于甘蔗，仙人掌，下界疣，紫颂植物等同样有效！",
+	"item.biomancy.gift_sac": "礼物肉囊",
+	"item.biomancy.gift_sac.tooltip": "薄薄的皮肤中好像包裹了些东西。要么是里面真装满了物品或者毒液，要么就是你的语言设置成了天书。\n\n手持肉囊右击以取出物品。",

```

说实话这里我没看懂。
Looks like some objects are wrapped in an organic layer of skin. Might be filled with items or toxin if your language is set to German.
if是作toxin 的补语还是这个句子的条件状语……

---

```suggestion
	"item.biomancy.gift_sac.tooltip": "薄薄的皮肤中好像包裹了些东西。里面也许真装满了物品，要是你的语言设置成了天书，里面是毒液也说不定。\n\n手持肉囊右击以取出物品。",
```
鉴定为在if前多读了个or导致的
我其实不太懂这里的German是什么奇怪的德语超长单词或者德语难学笑话，就直接叫天书了

---

add Gift-Sac item for reimbursing Items to the User when breaking changes happen that would lead to loss of progress
所以……就先这样吧，不用改了。看上去并不是什么掉落物